scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -110,10 +110,9 @@ workflow GvsExtractCallset {
                                                          else if GetNumSamplesLoaded.num_samples < 100000 then 20000 # Charlie
                                                               else 40000
 
-  Int effective_split_intervals_disk_size_override = if defined(split_intervals_disk_size_override) then select_first([split_intervals_disk_size_override])
-                                else if GetNumSamplesLoaded.num_samples < 100 then 50 # Quickstart
-                                     else 500
-
+Int effective_split_intervals_disk_size_override = select_first([split_intervals_disk_size_override, 
```

Comment:
i dont love this new spacing but 🤷‍♀️

Reply:
new run:
https://app.terra.bio/#workspaces/allofus-drc-wgs-dev/GVS%20AoU%20Echo%20RD/job_history/1aab2218-008f-4f3b-9520-9d763d0e4956